src/data/code-samples/en/input-errors/swift-ui.md

```diff
@@ -1,8 +1,6 @@
 # Input errors - SwiftUI
 
-In SwiftUI, we recommend using a [`Text`](https://developer.apple.com/documentation/swiftui/text) or [`Label`](https://developer.apple.com/documentation/swiftui/label) views to indicate an error. The error message should also be posted to assistive technologies by using an [`accessibility announcement`](../Techniques/accessibility-announcement.md).
-
-You could also use a third party library to displaying instructions. Unfortunately, accessibility is often not considered in the implementations.
+In SwiftUI, we recommend using [`Text`](https://developer.apple.com/documentation/swiftui/text) or [`Label`](https://developer.apple.com/documentation/swiftui/label) views to indicate an error. The error message should also be posted to assistive technologies by using an [`accessibility announcement`](../Techniques/accessibility-announcement.md).
```

Comment:
The URL on appt.org will be `https://appt.org/en/docs/swiftui/samples/input-errors`

Reply:
This also made me realize we are using `swift-ui.md` right now, looking at Apple docs and other websites, they are all using `swiftui`. Can you please replace `swift-ui.md` with `swiftui.md` everywhere. And also update it in the README and maybe other places where swift-ui is used.

Reply:
Link is now replaced with replaced with `https://appt.org/en/docs/ios/samples/accessibility-announcement`

Reply:
replaced `swift-ui` -> `swiftui` across the project